nautobot/extras/tables.py

```diff
@@ -1063,14 +1063,15 @@ def render_value(self, record):
 
 class NoteTable(BaseTable):
     actions = ButtonsColumn(Note)
-    created = tables.LinkColumn()
-    note = tables.Column(
+    created = tables.Column()
+    last_updated = tables.Column()
+    note = tables.LinkColumn(
```

Comment:
`LinkColumn` is deprecated, so perhaps:

```suggestion
    note = tables.Column(linkify=True)
```

However, given that the content of the note is rendered Markdown, I'm concerned about how this will look/behave given a more "realistic" set of test data, e.g a multi-paragraph note with inline Markdown formatting and maybe even hyperlinks of its own??

Reply:
Oh, that's an interesting point I hadn't considered.  It might be best to keep the "Created Date" column as the object link, but linkify it instead like you mentioned.  I'll test it out.